packages/react-provider/src/index.tsx

```diff
@@ -40,16 +41,18 @@ export const CourierProvider: React.FunctionComponent<ICourierContext> = ({
     return new Client({ clientKey, userId, userSignature, apiUrl });
   }, [clientKey, userId, userSignature, apiUrl]);
 
-  const courierTransport = useMemo(() => {
+  const transport = useMemo(() => {
```

Comment:
can we do something like this instead?  
```suggestion
  const transport = useMemo< CourierTransport >(() => {
```